services/grafana-loki/0.33.2/jobs.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: kustomize.toolkit.fluxcd.io/v1beta2
+kind: Kustomization
+metadata:
+  name: grafana-loki-jobs
+  namespace: ${releaseNamespace}
+spec:
```

Comment:
Does this need to depend on anything? e.g. does it need to be run after loki HR is up, or after minio is up? And I know not directly related to this PR but I saw `Tenant` which is a minio CR is defined in these charts as well, did we not run into issues of that potentially getting applied before the minio CRDs were actually created? Since I don't think we have any dependencies set there either 🤔  (should we set them?)

Reply:
This job won't succeed until after Minio is up and accepting requests. There's an inner retry loop that reattempts failed connections, and then the job itself will restart on failure. Not setting an explicit dependency between the job and Tenant hasn't caused any problems, so I don't see a reason to change that here.